projects/plugins/jetpack/class.jetpack-post-images.php

```diff
@@ -339,7 +356,7 @@ public static function from_thumbnail( $post_id, $width = 200, $height = 200 ) {
 				// If wp_get_attachment_image_src returns false but we know that there should be an image that could be used.
 				// we try a bit harder and user the data that we have.
 				$thumb_post_data = get_post( $thumb );
-				$img_src         = array( $thumb_post_data->guid ?? null, $meta['width'], $meta['height'] );
+				$img_src         = array( $thumb_post_data->guid, $meta['width'], $meta['height'] );
```

Comment:
I think this is a left-over from a rebase conflict. Looks like it reverts a recent change from https://github.com/Automattic/jetpack/pull/35264/files. Probably wasn't intentional?